Dockerfile

```diff
@@ -6,36 +6,35 @@ EXPOSE 80
 EXPOSE 8000
 EXPOSE 5051
 
-# allowed environment variables
-# ENABLE_WEBSERVER=true to start the RESTful API server, should only be set on webserver
-# DEPLOY_DB to start the database container (for DC/OS use)
-# DEPLOY_LOGGING to start up the logstash system
-# DEPLOY_WEBSERVER to start the webserver - without this we will run a naked scheduler
-# INIT_DB to initialize the database (migrate, load, etc.)
-# LOAD_COUNTRY_DATA to load country borders fixture into the database (don't select this if you have custom country data)
-# LOGSTASH_DOCKER_IMAGE the name of the DOcker image for logstash
-# SCALE_DEBUG
+# recognized environment variables
+# CONFIG_URI
```

Comment:
I looked through the documentation and I could not find a environment variables that presently exposes this inside the container. I think we should table that for now, but I'd like to go there in the future.